conf/distro/rpb-sota.conf

```diff
@@ -0,0 +1,12 @@
+require conf/distro/include/rpb.inc
+require conf/distro/sota.conf.inc
+
+DISTRO_NAME = "Reference-Platform-Build-SOTA"
+
+INHERIT += "sota"
+DISTRO_FEATURES_append = " sota"
```

Comment:
if you include sota.conf.inc, then you don't need to duplicate the config manually... that's the whole point.